src/platform/src/core/utils/secureApiProxyClient.js

```
@@ -3,333 +3,274 @@ import { getSession } from 'next-auth/react';
+            const separator = config.url.includes('?') ? '&' : '?';
+            config.url = `${config.url}${separator}token=${cachedApiToken}`;
           }
           break;
-        }
+
         case AUTH_TYPES.JWT:
         case AUTH_TYPES.AUTO:
-        default: {
-          // Use JWT token from session in Authorization header
+        default:
           const jwtToken = await getJwtToken();
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)